src/coreclr/gcinfo/gcinfoencoder.cpp

```diff
@@ -909,97 +909,6 @@ void GcInfoEncoder::FinalizeSlotIds()
 #endif
 }
 
-#ifdef PARTIALLY_INTERRUPTIBLE_GC_SUPPORTED
-
-// tells whether a slot cannot contain an object reference
-// at call instruction or right after returning
-bool GcInfoEncoder::DoNotTrackInPartiallyInterruptible(GcSlotDesc &slotDesc)
```

Comment:
I think this was an optimization in the past to filter out volatile registers early. 
It also claims to filter out things that cannot possibly contain a ref - but why would that even come as an input? The filtering probably did not help much in this regard.

Now that we record everything that is alive, the filtering is pointless indeed.